README.md

```diff
@@ -8,11 +8,11 @@
 
 ## What is Jib?
 
-Jib is a tool for building container images for your Java applications.
+Jib is a tool for building container images for your Maven-based Java applications.
```

Comment:
I think we can leave it as is since we will be adding a Gradle plugin as well. Or do you think it might be confusing to users to find that it is only for Maven currently?

Reply:
Will you be able to add Gradle support relatively soon? If not, I think it's important to let them know that it only works with Maven as soon as possible.

Reply:
Okay, sounds good. Gradle plugin won't be added until the Maven plugin has been worked out more. How does changing "a tool" to "a Maven plugin" sound?

Reply:
So, the new description would be:
Jib is a Maven plugin for building container images for your Java applications.

Is that correct?

Reply:
Yep!